data/card_definitions.json

```diff
@@ -8844,7 +8844,7 @@
 	{
 		"id": "phonon_turningsatisfaction",
 		"type": "special",
-		"display_name": "Turning Satisfaction",
```

Comment:
holy crap wow. I guess just leave the id the same because it would be annoying to change it all over? lol though